Source/WebKit/UIProcess/ios/forms/WKFormSelectPicker.mm

```diff
@@ -160,9 +160,9 @@ - (instancetype)initWithView:(WKContentView *)view
 
     [self _setMagnifierEnabled:NO];
 ALLOW_DEPRECATED_DECLARATIONS_BEGIN
```

Comment:
@RSilicon do you mind removing these deprecation warning suppressors?

Reply:
Oops yes, I missed that

Reply:
> @RSilicon do you mind removing these deprecation warning suppressors?

Yeah. Sorry, I was asleep and didn't see this.